internal/view/live_view.go

```diff
@@ -145,6 +147,21 @@ func (v *LiveView) bindKeys() {
 	}
 }
 
+// pauseCmd is used for pausing the refreshing of data on config map and secrets
+func (v *LiveView) pauseCmd(evt *tcell.EventKey) *tcell.EventKey {
+	if v.paused {
+		v.Start()
+		v.app.Flash().Info("Auto-refresh is enabled")
```

Comment:
Why is this `Info` and the other `Infof`?

Reply:
Just an oversight. I was trying to think of a way to combine the `Flash` statements.

I can quickly fix that. Are there no other issues? You are okay with all the other wording and where I've placed the functions/code?

Reply:
I'm not the owner. I think `Toggle` is better (it's used in a bunch of places in k9s).

Otherwise, it seems good to me.